.devcontainer/default/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+// Reference: https://containers.dev/implementors/json_reference/
+{
+  "name": "SourceKit-LSP",
+  "dockerFile": "Dockerfile",
+
+  // Allow the processes in the container to attach a debugger
+  "capAdd": [ "SYS_PTRACE" ],
+  "securityOpt": [ "seccomp=unconfined" ],
+
+  "mounts": [
+    // Use a named volume for the build products for optimal performance (https://code.visualstudio.com/remote/advancedcontainers/improve-performance#_use-a-targeted-named-volume)
+    "source=${localWorkspaceFolderBasename}-build,target=${containerWorkspaceFolder}/.build,type=volume",
+    // Do the same for experimental background indexing
+    "source=${localWorkspaceFolderBasename}-index-build,target=${containerWorkspaceFolder}/.index-build,type=volume"
+  ],
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "sswg.swift-lang"
+      ],
+      "settings": {
+        "lldb.launch.expressions": "native",
```

Comment:
I don't know if this has changed, but when the extension does it for you it prompts you where to store the setting, with one of the option (I believe the default) being "Global" and the other being "Workspace". *Both* will conflict with the setting needed for opening the workspace locally on macOS, and setting it here has it end up in the "remote" settings which is the correct place for it. If this setting is not what the extension expect I believe it will still prompt you to change it.

Reply:
The extension prompts you but it'll have already prompted you outside of the devcontainer.

Reply:
Let’s remove it then. Fewer settings is always good.

Reply:
OK, for the record I'm not removing the setting below because it actually needs to be different inside the container vs out.

Reply:
Yes the setting below is required